views/error.pug

```diff
@@ -1,6 +1,7 @@
 extends layout
```

Comment:
Should this be layout.pug? I'm assuming the '.pug' is not required if it's working on your end

Reply:
Pug will assume that an extension is `.pug` if a file extension is not present. We could add `.pug` for clarity if you'd like though!